Calculator/Program.cs

```diff
@@ -37,22 +37,39 @@ public static double ResultofOperation(int num1, int num2)
     }
 
     class Program { 
-        static void Main(string[] args)
+        public static void Main()
         {
             Console.WriteLine("Welcome to Calculator!");
             // Enter number1
             Console.WriteLine("Type first number and press Enter");
-            int number1 = Convert.ToInt32(Console.ReadLine());
+            //int number1 = Convert.ToInt32(Console.ReadLine());
+            string number1 = Console.ReadLine();
+            bool isInteger = int.TryParse(number1, out int firstnumber);
+            while(isInteger == false)
+            {
+                Console.WriteLine("You enter incorrect value");
+                Console.WriteLine("Please enter integer number");
+                number1 = Console.ReadLine();
+                isInteger = int.TryParse(number1, out firstnumber);
+            }
 
             // Enter number2
             Console.WriteLine("Type second number and press Enter");
-            int number2 = Convert.ToInt32(Console.ReadLine());
+            string number2 = Console.ReadLine();
+            isInteger = int.TryParse(number2, out int secondnumber);
+            while (isInteger == false)
```

Comment:
Why I could see the same could couple of times ? Please move it to the separate function